packages-next/fields/src/types/relationship/views/cards/index.tsx

```diff
@@ -123,125 +141,153 @@ export function Cards({
-            <InlineEdit
-              list={foreignList}
-              fields={field.display.inlineEdit!.fields}
-              onSave={newItemGetter => {
-                setItems({
-                  ...items,
-                  [id]: newItemGetter,
-                });
-                const itemsBeingEdited = new Set(value.itemsBeingEdited);
-                itemsBeingEdited.delete(id);
-                onChange({
-                  ...value,
-                  itemsBeingEdited,
-                });
-              }}
-              selectedFields={selectedFields}
-              itemGetter={itemGetter}
-              onCancel={() => {
-                const itemsBeingEdited = new Set(value.itemsBeingEdited);
-                itemsBeingEdited.delete(id);
-                onChange({
-                  ...value,
-                  itemsBeingEdited,
-                });
-              }}
-            />
-          </CardContainer>
-        ) : (
-          <CardContainer mode="view" key={id}>
-            <Stack gap="xlarge">
-              {field.display.cardFields.map(fieldPath => {
-                const field = foreignList.fields[fieldPath];
-                const itemForField: Record<string, any> = {};
-                for (const graphqlField of getRootGraphQLFieldsFromFieldController(
-                  field.controller
-                )) {
-                  const fieldGetter = itemGetter.get(graphqlField);
-                  if (fieldGetter.errors) {
-                    const errorMessage = fieldGetter.errors[0].message;
-                    return (
-                      <FieldContainer>
-                        <FieldLabel>{field.label}</FieldLabel>
-                        {errorMessage}
-                      </FieldContainer>
-                    );
-                  }
-                  itemForField[graphqlField] = fieldGetter.data;
-                }
-
-                return (
-                  <field.views.CardValue
-                    key={fieldPath}
-                    field={field.controller}
-                    item={itemForField}
-                  />
-                );
-              })}
-            </Stack>
-            <Stack across gap="small" marginTop="xlarge">
-              {field.display.inlineEdit && onChange !== undefined && (
-                <Button
-                  size="small"
-                  disabled={onChange === undefined}
-                  onClick={() => {
-                    onChange({
+      <Stack
+        as="ul"
+        gap="xlarge"
+        css={{
+          padding: 0,
+          marginBottom: 0,
+          li: {
+            listStyle: 'none',
+          },
+        }}
+      >
+        {[...value.currentIds].map((id, index) => {
+          const itemGetter = items[id];
+          const isEditMode = !!(onChange !== undefined) && value.itemsBeingEdited.has(id);
+          return (
+            <CardContainer role="status" mode={isEditMode ? 'edit' : 'view'}>
```

Comment:
persistent card container now so we can append a role status, and announce when a card enters edit mode and vice versa.